app/javascript/service/router.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+import Home from './views/Home.vue';
+
+Vue.use(Router);
+
+export default new Router({
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: Home,
```

Comment:
Isn't it better to rename this to Service Response Collection or something similar? Home is too generic and is used in multiple places.